ci-operator/jobs/openshift/release/openshift-release-release-4.3-periodics.yaml

```diff
@@ -757,6 +758,25 @@ periodics:
 
         # debug output
         oc describe -n ${namespace} istag/machine-os-content:latest
+
+        # Craft a release image
+        MOC_SHA256=$( oc get istag -n "${namespace}" istag/machine-os-content:latest -o template='{{ .image.metadata.name }}' )
+        export RELEASE_IMAGE_LATEST=registry.svc.ci.openshift.org/${namespace}/release:latest
+        export IMAGE_TESTS=registry.svc.ci.openshift.org/origin/$(BRANCH):tests
+        oc registry login
+        oc adm release new --from-release=registry.svc.ci.openshift.org/origin/release:$(BRANCH) \
```

Comment:
You need to use the image stream as the source.  This would create from too old of a branch.  We should be testing against head of each image, which requires from image stream.

Reply:
Updated to `-n origin --from-image-stream=$(BRANCH)`